lib/ecto/migration/schema_migration.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Ecto.Migration.SchemaMigration do
-  # Define a schema that works with the a table, which is schema_migrations by default
-  @moduledoc false
```

Comment:
I'm not sure if we should publicly document this module.

Reply:
Sure; I'll update the original comment instead of turning it into a moduledoc.